src/Utilities/ProgressReportEngine.cpp

```diff
@@ -19,18 +19,16 @@ namespace qmcplusplus
 {
 void ReportEngine::echo(xmlNodePtr cur, bool recursive)
 {
-  if (cur == NULL)
+  if (cur == nullptr)
     return;
-  app_debug() << "<input node=\"" << (const char*)(cur->name) << "\"";
+  app_debug() << R"(<input node=")" << (const char*)(cur->name) << '"';
   xmlAttrPtr att = cur->properties;
-  char atext[1024];
-  while (att != NULL)
+  while (att != nullptr)
   {
-    sprintf(atext, "  %s=\"%s\"", (const char*)(att->name), (const char*)(att->children->content));
-    app_debug() << atext;
+    app_debug() << "  " << (const char*)(att->name) << R"(=")" << (const char*)(att->children->content) << '"';
     att = att->next;
   }
-  app_debug() << "/>\n";
```

Comment:
I think this is a forward slash marking the end of an XML tag, not a backslash escaping the next character.    But since this is just output to app_debug, and not an actual XML file, it doesn't really matter.
In a larger context, it's annoying when the code produces XML on stdout/stderr,  and a better fix would be to change the output message so it's not XML.  Not a change for this PR, though.

Reply:
Fixed.